site/posts/intake-esm-2020-3-16.ipynb

```diff
@@ -0,0 +1,492 @@
+    " \n",
+    "\n",
+    "```python\n",
+    "import intake\n",
+    "col = intake.open_esm_datastore(\"sample-collection.json\")\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Thanks to [Joe Hamman](https://github.com/jhamman) for proposing this feature and reviewing the implementation. Thanks to [Brian Bonnlander](https://github.com/bonnland) for implementing this feature. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Relative paths for catalog files\n",
+    "\n",
+    "Fetching and loading catalog files in earlier version of `intake-esm` required using absolute paths/urls for the catalog file (CSV). \n",
+    "\n",
+    "For example:\n",
+    "\n",
+    "\n",
+    "- `old_sample.json`: \n",
+    "```json\n",
+    "{\n",
+    "  \"esmcat_version\": \"0.1.0\",\n",
+    "  \"id\": \"campaign-cesm2-cmip6-timeseries\",\n",
+    "  \"description\": \"ESM collection for the CESM2 raw output that went into CMIP6 data. Located in campaign storage, accessible via GLADE on casper\",\n",
+    "  \"catalog_file\": \"/glade/collections/cmip/catalog/intake-esm-datastore/catalogs/campaign-cesm2-cmip6-timeseries.csv.gz\",\n",
+    "    ...\n",
+    "}\n",
+    "```\n",
+    "    \n",
+    "\n",
+    "Today  the `catalog_file` can point to a full path or a path relative to the input JSON file path:\n",
+    "\n",
+    "\n",
+    "- `new_sample.json`:\n",
+    "```json\n",
+    "{\n",
+    "  \"esmcat_version\": \"0.1.0\",\n",
+    "  \"id\": \"campaign-cesm2-cmip6-timeseries\",\n",
+    "  \"description\": \"ESM collection for the CESM2 raw output that went into CMIP6 data. Located in campaign storage, accessible via GLADE on casper\",\n",
+    "  \"catalog_file\": \"campaign-cesm2-cmip6-timeseries.csv.gz\",\n",
+    "    ...\n",
+    "}\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Acknowledgements\n",
+    "\n",
+    "The following people contributed to the [NCAR/intake-esm](https://github.com/NCAR/intake-esm), [NCAR/esm-collection-spec](https://github.com/NCAR/esm-collection-spec) repositories since `intake-esm` release `2019.12.13` on December 13th, 2019:\n",
+    "\n",
+    "\n",
+    "- Anderson Banihirwe\n",
+    "- Brian Bonnlander\n",
+    "- Joe Hamman\n",
+    "- Julius Busecke"
+   ]
+  }
+ ],
+ "metadata": {
+  "kernelspec": {
+   "display_name": "Python [conda env:analysis]",
+   "language": "python",
+   "name": "conda-env-analysis-py"
+  },
+  "language_info": {
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.7.6"
+  },
+  "nikola": {
+   "author": "Anderson Banihirwe",
+   "category": "data-catalog",
+   "date": "2020-03-16 19:27:53 UTC-06:00",
```

Comment:
Should I changed this to today's date?

Reply:
You can, but it's not necessary.  Since this PR hasn't merged, yet, you might as well.